packages/radio-group/test/lion-radio-group.test.js

```diff
@@ -11,6 +10,41 @@ import '../lion-radio.js';
 const fixture = /** @type {(arg: TemplateResult) => Promise<LionRadioGroup>} */ (_fixture);
 
 describe('<lion-radio-group>', () => {
+  it('restores to empty modelValue if changes were made', async () => {
```

Comment:
Should these tests be moved to the ChoiceGroupMixin suite? I think these tests apply for all single choice groups right?

Reply:
Only the reset code is not part of the `ChoiceGroupMixin`, but the `FormGroupMixin`, should be applied there. Only the `FormGroupMixin-suite` as is, can't be used in other components. It needs a big refactor to make it suitable.

Will add a TODO